src/SFA.DAS.EmployerAccounts.Web/Controllers/OrganisationController.cs

```diff
@@ -85,31 +86,26 @@ public async Task<ActionResult> OrganisationAddedNextStepsSearch(string organisa
 
             var response = await _orchestrator.CreateLegalEntity(request);
 
+            if (response.Status == HttpStatusCode.Unauthorized)
+            {
+                return View(ControllerConstants.AccessDeniedViewName);
```

Comment:
If you just return `View(response)` then the BaseController should handle the response and redirect, also adding the accountId to the ViewBag

Reply:
Done